src/main/java/com/dabolink/XPRelicSoundConfig.java

```diff
@@ -10,17 +10,17 @@ public interface XPRelicSoundConfig extends Config
 	@ConfigItem(
 		keyName = "relic sound chance",
 		name = "1/x chance (per xp)",
-		description = "1 in x chance per xp for relic sound to play"
+		description = "1 in x chance per xp for relic sound to play."
 	)
 	default int procChance()
 	{
 		return 100000;
 	}
 
 	@ConfigItem(
-		keyName = "include maxed skills",
-		name = "include maxed skills",
-		description = "if true, roll even if the skill is maxed"
+		keyName = "Include maxed skills.",
```

Comment:
I don't think the key for the config is supposed to have punctuation

Reply:
Should be resolved.